src/pages/test-results-search/test-results-search.ts

```diff
@@ -74,7 +78,10 @@ export class TestResultsSearchPage extends BasePageComponent {
             this.searchResults = results;
             this.showSearchSpinner = false;
           }),
-          catchError(() => {
+          catchError((err) => {
+            const log: Log = this.logHelper
+              .createLog(LogType.ERROR, `Seaching tests by driver number (${this.candidateInfo})`, err);
```

Comment:
Could be worth checking that we are allowed to store PII in the logging service. As the Driver Number is personal information